api4/user.go

```diff
@@ -725,6 +725,15 @@ func updateUserActive(c *Context, w http.ResponseWriter, r *http.Request) {
 		c.Err = err
 	} else {
 		c.LogAuditWithUserId(user.Id, fmt.Sprintf("active=%v", active))
+		if isSelfDeactive {
+			c.App.Go(func() {
+				if !active {
```

Comment:
If `isSelfDeactive` is true, then `active` must be false, so I think this check is not needed.